launch_ros/launch_ros/utilities/to_parameters_list.py

```diff
@@ -45,8 +61,26 @@ def to_parameters_list(
     for params_set_or_path in evaluated_parameters:
         if isinstance(params_set_or_path, pathlib.Path):
             with open(str(params_set_or_path), 'r') as f:
+                node_name.lstrip('/')
+                if namespace:
+                    namespace.lstrip('/')
+                    node_name = f"{namespace}/{node_name}"
+                param_dict = yaml.safe_load(f)
+
+                # Get all keys that come before "ros__parameters"
+                keys = []
+                inner_dict = {}
+                (keys, inner_dict) = search_for_key(keys, param_dict)
```

Comment:
@rebecca-butler nit: parenthesis are redundant when unpacking:
```suggestion
                keys, inner_dict = search_for_key(keys, param_dict)
```